lib/travis/api/v3/queries/user.rb

```diff
@@ -1,7 +1,7 @@
 module Travis::API::V3
   class Queries::User < Query
     setup_sidekiq(:user_sync, queue: :sync, class_name: "Travis::GithubSync::Worker")
-    params :id, :login, :email, :github_id, :is_syncing
+    params :id, :login, :email, :github_id, :is_syncing, :github_installation_id
```

Comment:
This should no longer be necessary. With the changes, the owner should be retrieved by fetching the installation first (ie, thus using the installation query).